specs/proposals.md

```diff
@@ -88,10 +80,49 @@ either via fault proof or by an attestation proof, then the bond can be slashed
 payment to the users who paid for gas to remove the malicious output.
 
 The `op-proposer` can still be used to submit output proposals. A naive implementation of the
-`op-proposer` will still submit output proposals on an interval. A more ideal implementation
+`op-proposer` will submit output proposals on an interval. However, this is not required, and other
+proposer implementations may submit valid outputs at any time. A more ideal implementation
 will use heuristics such as time of last submission or number of pending withdrawals that have
 yet to be included in an output proposal.
 
+In order to post outputs to the `L2OutputOracle`, the proposer needs to periodically query the
+[rollup node](./rollup-node.md)'s [`optimism_outputAtBlock` rpc method](./rollup-node.md#l2-output-rpc-method)
```

Comment:
Slight nit because this is under the v2 section but applies to both versions, perhaps we can move common stuff to the top so its not under a particular section or add a new section for this